web/util/send_email.py

```diff
@@ -0,0 +1,20 @@
+from flask import current_app
+from sendgrid import SendGridAPIClient
+from sendgrid.helpers.mail import Mail
+
+
+def send_email(to_emails, template_id, template_data):
+    msg = Mail(
+        from_email="info@gridt.org", to_emails=to_emails
+    )
+    
+    msg.template_id = template_id
+    msg.template_data = template_data
+
+    try:
+        sg = SendGridAPIClient(current_app.config["EMAIL_API_KEY"])
+        resp = sg.send(msg)
+        return resp.status_code, resp.body, resp.headers
+    except Exception as e:
+        if e.message:
+            return e.message
```

Comment:
I think this error catching is dangerous. If something is going wrong we want to know, loud and clear, both on the diagnostics and the client side. You are also not really doing anything with the returned information later on, so I think it would be best just to remove the catch clause.